app/views/admin_request/show.html.erb

```diff
@@ -372,6 +372,11 @@
 
 <hr>
 
+<h2> Attachments </h2>
```

Comment:
I'd add a little help block here explaining that we're excluding the main body attachments:

```html
<h2>Attachments</h2>

<div class="span12 help-block">
  <p>Excludes main body part attachments.</p>
</div>
```